### PR TITLE
Added OS detecting capabilities

### PR DESCRIPTION
**Pull request by yu-eric**
File can now detect whether the user is on OSX or Linux, and grab the correct usearch file

### CODE REVIEW

File: flask/cluster.py [hunk 1]
```diff
@@ -2,11 +2,15 @@
...
+from sys import platform
 
 
 uclust_identity = utils.get_config()['uclust_identity'] # how similar sequences in the same cluster must be
 sequences_filename = 'usearch/sequences.fsa'
-usearch_binary_filename = 'usearch/usearch10.0.240_i86linux32'
+if platform == "linux" or platform == "linux2":
+    usearch_binary_filename = 'usearch/usearch10.0.240_i86linux32'
+elif platform == "darwin":
+    usearch_binary_filename = 'usearch/usearch11.0.667_i86osx32'
```
Comment (michael13162) [inline flask/cluster.py]: How should we handle when platform doesn't equal any of these strings?  Maybe add an else that reports the error?

Comment (yu-eric) [reply flask/cluster.py]: See ce785166be8d97a4894ab25ed46bec244b77d12d

**Comment by michael13162** [comment]
LGTM

**Comment by cjmyers** [comment]
Me too

**Pull request merged**